### ISSUE TITLE
/health/ready fails when migrations are done using non-Kratos migration tech stack

### ISSUE THREAD

**Issue by tristanls** [ory/kratos#2023]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/kratos/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/kratos/blob/master/CONTRIBUTING.md).
- [ ] This issue affects my [Ory Cloud](https://www.ory.sh/) project.
- [ ] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [ ] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe the bug

Our migration tool is `gomigrate`. In order not to introduce yet-another-migration technology, we process kratos migration files and apply them using our migration tech stack.

All would be well, however `/health/ready` attempts to read migration status.

https://github.com/ory/kratos/blob/9fa79861130809e34add84ec43ec95964b17124e/driver/registry_default.go#L218-L229

which fails.

### Reproducing the bug

Use different `schema_migrations` table schema for migrations.

### Version

v0.8.0-alpha.3

### Additional Context

The ask here is to either get rid of migrations in readiness check or a cli/config param to turn off this portion of readiness check.

https://github.com/ory/kratos/blob/9fa79861130809e34add84ec43ec95964b17124e/driver/registry_default.go#L218-L229

**Comment by tristanls** [comment]
We're gonna use migrations tech that comes with Kratos. Feel free to disregard at this point.

**Comment by aeneasr** [comment]
I don’t think we can support other migration stacks, please use the one provided. It works well and has tons of tests ensuring that migrations works without issues. If you’re ok with it I’ll close this one as wontfix

**Comment by aeneasr** [comment]
I don’t think we can support other migration stacks, please use the one provided. It works well and has tons of tests ensuring that migrations works without issues. If you’re ok with it I’ll close this one as wontfix
